src/spares/Apps/oc_capture.lsl

```diff
@@ -366,12 +371,22 @@ default
                 g_iExpireMode=0;
             }
         }
+        
+        if (g_iCaptured && llGetAgentSize(g_kCaptor) == ZERO_VECTOR){
+            g_iReleaseTicks--;
+            if (g_iReleaseTicks == 599) llMessageLinked(LINK_SET,NOTIFY, "0You are out of range! Capture of secondlife:///app/agent/"+(string)g_kWearer+"/about will end in 10 minutes unless you come back!", g_kCaptor);
```

Comment:
This would be more reliable if you use Unix time. Lag can affect the update of variables and event ticks. Just simply check if the person has left, and set the Unix time to : llGetUnixTime()+(10*60). That is also when you can send the notify.

Reply:
are you sure the time must be that precise? if its 9 or 11 minutes it should be fine

Reply:
In a lagged region. This can be extended by lag up to an hour. I have also seen one hour times that used this method take weeks to resolve when staying in the same lagged region :)

Reply:
ok i'll change that later :-)

Reply:
:)